hokusai/development.yml

```diff
@@ -0,0 +1,40 @@
+---
+version: '2'
+services:
+  force:
+    command: yarn start
+    environment:
+    - NODE_ENV=development
```

Comment:
I wonder if there's a good way to avoid this getting out of sync with .env.oss. However, maybe the idea is eventually we'd just use hokusai and ditch the .env.oss files.